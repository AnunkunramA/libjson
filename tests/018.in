{ foo : 42, // comment until end-of-line
  bar : 99 }
