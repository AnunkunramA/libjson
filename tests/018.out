{"foo":42, "bar":99}
